apps/root/src/pages/aggregator/swap-container/components/landing/index.tsx

```diff
@@ -1,209 +1,166 @@
 import * as React from 'react';
-import { Typography, Chip, Paper, CheckCircleOutlineOutlinedIcon, colors } from 'ui-library';
-import EmptyRoutes from '@assets/svg/emptyRoutes';
-import { FormattedMessage } from 'react-intl';
-import { withStyles } from 'tss-react/mui';
-import compact from 'lodash/compact';
-import styled from 'styled-components';
+import {
+  Typography,
+  BalmyLogoSmallDark,
+  BalmyLogoSmallLight,
+  BackgroundPaper,
+  ContainerBox,
+  TickCircleIcon,
+  useTheme,
+  colors,
+  BackgroundGrid,
+} from 'ui-library';
+import { FormattedMessage, defineMessages, useIntl } from 'react-intl';
+import styled, { keyframes } from 'styled-components';
+import useSdkDexes from '@hooks/useSdkSources';
+import { chunk } from 'lodash';
 import TokenIcon from '@common/components/token-icon';
 import { emptyTokenWithLogoURI } from '@common/utils/currency';
-import useSdkDexes from '@hooks/useSdkSources';
-import MinimalTabs from '@common/components/minimal-tabs';
-import { SourceMetadata } from '@mean-finance/sdk';
-import AggregatorFAQ from '../faq';
-
-const StatusChip = withStyles(Chip, ({ palette: { mode } }) => ({
-  colorSuccess: {
-    background: colors[mode].semanticBackground.success,
-    color: colors[mode].semantic.success.primary,
-  },
-  colorError: {
-    background: colors[mode].semanticBackground.error,
-    color: colors[mode].semantic.error.primary,
-  },
-}));
 
-const StyledPaper = styled(Paper)<{ $column?: boolean; $align?: boolean }>`
-  padding: 16px;
+const BulletPoint = ({ label }: { label: string }) => (
+  <ContainerBox gap={1} alignItems="center">
+    <TickCircleIcon color="primary" />
+    <Typography variant="body2" fontWeight={700} noWrap>
+      {label}
+    </Typography>
+  </ContainerBox>
+);
+
+const StyledBackgroundPaper = styled(BackgroundPaper)`
+  ${({ theme: { spacing } }) => `
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing(6)};
+  align-items: center;
+  padding-top: ${spacing(12)};
+  padding-bottom: ${spacing(47.5)};
   position: relative;
   overflow: hidden;
-  border-radius: 20px;
-  flex-grow: 1;
-  display: flex;
-  gap: 24px;
-  flex-direction: ${({ $column }) => ($column ? 'column' : 'row')};
-  ${({ $align }) => ($align ? 'align-self: flex-start;' : '')}
+`}
 `;
 
-const StyledCenteredWrapper = styled.div`
-  display: flex;
-  flex: 1;
-  align-items: center;
-  justify-content: center;
-  flex-direction: column;
-  gap: 10px;
+const StyledBottomBackground = styled.div`
```

Comment:
shouldnt this be a paper?

Reply:
Not sure, maybe for some good practice? Paper doesn't provide any useful property in this case, in fact it would apply boder-radius that is not needed here. This is just a box with a fancy violet background